python/amici/pysb_import.py

```diff
@@ -338,9 +339,9 @@ def _get_sigma_name_and_value(
     if obs_name in sigmas:
         if sigmas[obs_name] not in pysb_model.expressions:
             raise ValueError(f'value of sigma {obs_name} is not a '
-                            f'valid expression.')
-        sigma_name = pysb_model.expressions[sigmas[obs_name]].name
-        sigma_value = pysb_model.expressions[sigmas[obs_name]].expand_expr()
+                             f'valid expression.')
+        sigma_name = pysb_model.expressions[sigmas[obs_name].name].name
```

Comment:
Cleanup after clarifying intended usage

Reply:
@FFroehlich : Here things didn't work as documented. Some lines assumed sigmas were provided as `pysb.Expression`, others as `pysb.Expression.name` (`str`). Documentation claims the latter. Seems it isn't tested anywhere. I will then adapt to using `str`?!

Reply:
Yes, I think its more user-friendly to require names (`str`) here.

Reply:
Agreed. Done.